components/mysql_ssl/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/mysql_ssl",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Not sure if this is the case, but if users have to reconnect their account, it might warrant a new major version (1.0.0)